web3rpc/rpc-specs/paths/governance/getRewardsAccumulated.yaml

```diff
@@ -65,7 +67,7 @@ paths:
 
       responses:
         200:
-          description: Accumulated reward information in a given block range
+          description: Returns the accumulated reward distribution details.
```

Comment:
1. Change the type of response.firstBlockTime and response.lastBlockTime to integers. We changed their format from human-readable string to unix timsteamp.
2. Change the example response as well.